e2e/select/select.chromium.e2e-spec.ts

```diff
@@ -0,0 +1,16 @@
+import {SelectPO} from '@agnos-ui/page-objects';
+import {expect, test} from '../fixture';
+import {analyze} from '../utils';
+
+test.describe('Accessibility', () => {
+	test('Opened select', async ({page, browserName}) => {
+		if (browserName === 'chromium') {
+			await page.goto('#/select/default');
+			const selectPO = new SelectPO(page);
+			const locatorInput = selectPO.locatorInput;
+			await locatorInput.fill('a');
+			await locatorInput.press('Enter');
```

Comment:
maybe add an expect before the axe tests to make sure the popup is opened.

Reply:
Done ;). I've added more by the way:

- the first one to be sure the first item is selected,
- the second one to be sure that the dropdown is still open.